step-controller-plugins/step-controller-plugins-timeseries/src/main/java/step/plugins/timeseries/dashboards/model/ChartFilterItem.java

```diff
@@ -0,0 +1,104 @@
+package step.plugins.timeseries.dashboards.model;
+
+import jakarta.validation.constraints.NotNull;
+
+import java.util.List;
+
+public class ChartFilterItem {
```

Comment:
For all classes with Chart in their name, are they restricted to chart or apply to dashblet. In the later case, they should be renamed

Reply:
@david-stephan they are used in dashlets, but also in the dashboard (the global filters). maybe call them TimeSeriesFilterItem?

Reply:
@lucianbaciu ultimetaley global filters are used by dashlets too, but TimeSeriesFilterItem is fine with me.